Header header

bool[13] LOCK
bool[13] RD
bool[13] RG
